samcli/lib/build/app_builder.py

```diff
@@ -611,3 +629,73 @@ def _parse_builder_response(stdout_data: str, image_name: str) -> Dict:
             raise ValueError(msg)
 
         return cast(Dict, response)
+
+    @staticmethod
+    def _get_env_vars_value(filename: Optional[str]) -> Any:
```

Comment:
The docstring writes return value is a `dict`, can we use `Dict` here instead of `Any`?

> :return dict: Value of environment variables, if provided. None otherwise

Reply:
But the type check would fail when we return none if we type hint this to be Dict

Reply:
`Optional[Dict]` can be used if it can be None

Reply:
and `json.loads()` returns `Any`, since we know it is a JSON dictionary, we can do `cast(Dict, parsed_json)`

Reply:
sure let me try this change with tests!

Reply:
Changed the default return value from None to empty dict, got rid of the Any return type, otherwise it always requires it to be Any